### ISSUE TITLE
[Bug] MixinProperties (possibly) causing a really cryptic crash

### ISSUE THREAD

**Issue by oliviathevampire** [TerraformersMC/Terraform#88]
When I use newest alpha version of Terraform I get this crash 
```
[19:43:44] [main/WARN] (Minecraft) Failed to get system info for GL Caps
 java.lang.IllegalStateException: Rendersystem called from wrong thread
	at com.mojang.blaze3d.systems.RenderSystem.constructThreadException(RenderSystem.java:183) ~[minecraft-merged-b12d2109a1-24w06a-loom.mappings.24w06a.layered+hash.209589215-v2.jar:?]
	at com.mojang.blaze3d.systems.RenderSystem.assertOnRenderThread(RenderSystem.java:172) ~[minecraft-merged-b12d2109a1-24w06a-loom.mappings.24w06a.layered+hash.209589215-v2.jar:?]
	at com.mojang.blaze3d.systems.RenderSystem.getCapsString(RenderSystem.java:872) ~[minecraft-merged-b12d2109a1-24w06a-loom.mappings.24w06a.layered+hash.209589215-v2.jar:?]
	at net.minecraft.SystemReport.setDetail(SystemReport.java:66) ~[minecraft-merged-b12d2109a1-24w06a-loom.mappings.24w06a.layered+hash.209589215-v2.jar:?]
	at net.minecraft.client.Minecraft.fillSystemReport(Minecraft.java:2617) ~[minecraft-merged-b12d2109a1-24w06a-loom.mappings.24w06a.layered+hash.209589215-v2.jar:?]
	at net.minecraft.client.Minecraft.fillReport(Minecraft.java:2588) ~[minecraft-merged-b12d2109a1-24w06a-loom.mappings.24w06a.layered+hash.209589215-v2.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:178) ~[minecraft-merged-b12d2109a1-24w06a-loom.mappings.24w06a.layered+hash.209589215-v2.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.6.jar:?]
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86) ~[dev-launch-injector-0.2.1+build.8.jar:?]
---- Minecraft Crash Report ----
// I bet Cylons wouldn't have this problem.

Time: 2024-02-09 19:43:44
Description: Bootstrap

java.lang.NoSuchMethodError: 'net.minecraft.world.level.block.state.properties.IntegerProperty net.minecraft.world.level.block.state.properties.IntegerProperty.of(java.lang.String, int, int)'
	at net.minecraft.world.level.block.state.properties.BlockStateProperties.<clinit>(BlockStateProperties.java:165)
	at net.minecraft.world.level.block.FireBlock.<clinit>(FireBlock.java:43)
	at net.minecraft.server.Bootstrap.bootStrap(Bootstrap.java:54)
	at net.minecraft.client.main.Main.main(Main.java:170)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86)
```

**Comment by gniftygnome** [comment]
The main concern I have about this is going to be reproducing it, since it didn't occur in testing.  I'll see what I can do once I have a moment.

**Comment by gniftygnome** [comment]
So...  I am wondering now if this problem is part of a complex of remapping issues with Loom 1.5.  It looks very similar.

https://github.com/FabricMC/fabric-loom/issues/1044
https://github.com/architectury/architectury-loom/issues/194

**Comment by gniftygnome** [comment]
I asked about this in Loom's issue tracker and Modmuss responded by asking, "what loader version is being used, has the mod been runtime remapped or via loom?"  Also, can you try running the `clean` build task, killing your gradle daemon, deleting your loom cache, and then running `gradle --refresh-dependencies` and retrying the build?

**Comment by gniftygnome** [comment]
> [architectury/architectury-loom#194](https://github.com/architectury/architectury-loom/issues/194)

I believe I have eliminated the arch loom issue from this group of problems.  The arch loom issue does not reproduce if I convert Biolith back to a fabric loom project.


Regarding the original issue here, though, I remain unable to reproduce.  Please provide a full copy of the log, and if possible, a link to the project source which fails to build.

**Comment by gniftygnome** [comment]
Resolved in `10.0.0-alpha.6` released today.
